core/common/src/main/java/alluxio/metrics/Metric.java

```diff
@@ -321,26 +277,15 @@ public static Metric from(String fullName, double value, MetricType metricType)
     Preconditions.checkArgument(pieces.length > 1, "Incorrect metrics name: %s.", fullName);
     int len = pieces.length;
     String hostname = null;
-    String id = null;
-    String name = null;
     int tagEndIndex = len;
     // Master or cluster metrics don't have hostname included.
-    if (pieces[0].equals(MetricsSystem.InstanceType.MASTER.toString())
-        || pieces[0].equals(MetricsSystem.CLUSTER)) {
-      name = pieces[1];
-    } else {
-      if (pieces[len - 1].contains(ID_SEPARATOR)) {
-        String[] ids = pieces[len - 1].split(ID_SEPARATOR);
-        hostname = ids[0];
-        id = ids[1];
-      } else {
-        hostname = pieces[len - 1];
-      }
-      name = pieces[1];
+    if (!pieces[0].equals(MetricsSystem.InstanceType.MASTER.toString())
+        && !pieces[0].equals(MetricsSystem.CLUSTER)) {
+      hostname = pieces[len - 1];
       tagEndIndex = len - 1;
     }
     MetricsSystem.InstanceType instance = MetricsSystem.InstanceType.fromString(pieces[0]);
-    Metric metric = new Metric(instance, hostname, id, metricType, name, value);
+    Metric metric = new Metric(instance, hostname, metricType, pieces[1], value);
```

Comment:
`pieces[1]` seems kind of ambiguous compared to the previous `name` variable. Can we add a comment on what it is, or use the old variable?